W-17506689 flex lifecycle mb (#752)
mulesoft/docs-gateway: gateway/1.9/modules/ROOT/pages/flex-gateway-version-lifecycle.adoc

= Flex Gateway Version Lifecycle
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../assets/images

== Edge and LTS Release Channels

Flex Gateway releases are available in two release channels, Edge and Long-term Support (LTS). 

[%header,cols="4*a"]
|===

| Channel Type | Release Cadence | Differentiators | Support Coverage

| *Edge* 
| Three times per year 

> | Includes new features more often, and has a shorter support period for each release.

Reviewer: ```suggestion
| Includes new features more often and has a shorter support period for each release.
```

a| 
* 4 months of Standard Support
* 4 months of Extended Support

| *LTS*
| Once per year

> | Bundles new features in an annual release, and has a longer support period for each release.

Reviewer: ```suggestion
| Bundles new features in an annual release and has a longer support period for each release.
```

a|
* 12 months of Standard Support
* 12 months of Extended Support

|===

> Consider the following factors to determine which release channel is the best choice for your requirements.

Reviewer: ```suggestion
Consider these factors to determine which release channel is the best choice for your requirements.
```

* Edge release channel
+ 

> Mulesoft releases Flex Gateway Edge versions three times per year. Use the Edge channel if you want to stay on top of the latest features and feel comfortable with a more frequent release cycle and auto-upgrade cycle. The Edge channel includes the latest features and innovations available on the MuleSoft platform.

Reviewer: ```suggestion
MuleSoft releases Flex Gateway Edge versions three times per year. Use the Edge channel if you want to stay on top of the latest features and feel comfortable with a more frequent release cycle and auto-upgrade cycle. The Edge channel includes the latest features and innovations available on the MuleSoft platform.
```

* LTS release channel
+ 

> MuleSoft releases Flex Gateway Long-Term Support (LTS) versions once a year. Use the LTS channel if you want to avoid upgrading to a new version frequently. The LTS channel adopts new capabilities after they have been introduced via Edge releases.

Reviewer: ```suggestion
MuleSoft releases Flex Gateway Long-Term Support (LTS) versions once a year. Use the LTS channel if you want to avoid upgrading to a new version frequently. The LTS channel adopts new capabilities after they've been introduced via Edge releases.
```

> The following table shows the length of support for Flex Gateway releases:

Reviewer: ```suggestion
This table shows the length of support for Flex Gateway releases:
```

[%header%autowidth.spread]
|===
| Release Channel | Gateway Version |Release Date | End of Standard Support | End of Extended Support
| Edge | 1.9 | Mar 2025 | Jul 2025 | Nov 2025
| Edge | 1.10 | Jul 2025 | Nov 2025 | Mar 2026
| Edge | 1.11 | Nov 2025 | Mar 2026 | Jul 2026
| Edge | 1.12 | Mar 2026 | Jul 2026 | Nov 2026
| LTS | 1.9 | Mar 2025 | Mar 2026 | Mar 2027
| LTS | 1.12 | Mar 2026 | Mar 2027 | Mar 2028
|===

> In March, both Edge and LTS release with the same underlying version.

Reviewer: ```suggestion
In March, the Edge and LTS channels release with the same underlying version.
```

> For more information about the support policy that aligns with the release periods and types of releases, see the https://www.salesforce.com/content/dam/web/en_us/www/documents/legal/Agreements/versioning-back-support-policy.pdf[MuleSoft Support Policy^]. 

Reviewer: ```suggestion
For more information about the support policy that aligns with the release periods and types of releases, see the https://www.salesforce.com/content/dam/web/en_us/www/documents/legal/Agreements/versioning-back-support-policy.pdf[MuleSoft Support Policy]. 
```
